### DISCUSSION TITLE
https://adyanth.site/about/

### DISCUSSION THREAD

**Discussion by giscus** [adyanth/blog-comments#15] [General]
# https://adyanth.site/about/

Hi there!
I am Adyanth, pleased to meet you! I am your typical tech enthusiast/geek. Whatever label that works. What is not so run-of-the-mill about me is that I don’t tend to limit my adventures to any particular domain. I’ll usually be interested in anything technical, computers or otherwise.
 I am currently working at Cisco Systems  as a network security engineer. Titles aside, I mainly work on creating proof of concepts, new tools, and applications, generating services and opportunities for Cisco in the security domain.

https://adyanth.site/about/

**Comment by Anuragks9740267432** [comment]
This is amazing Adyanth! I hope this grows to great heights!

**Reply by adyanth** [reply]
Thanks Anurag!

**Comment by karthikpandelu** [comment]
Great work Adyanth! All the best.
